src/deploy-builder.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path'
+import * as fs from 'fs-p'
+
+export class DeployBuilder {
```

Comment:
Sure. It does :)
The purpose of this class is to prepare the deployment. It just happens that I refactored a little piece of it. I intended to put all the deployment part on it.
What do you think? Should I complete the refactoring or keep it in the monolite way?